### ISSUE TITLE
Locks control

### ISSUE THREAD

**Issue by Calzor-Suzay** [thomas-villagers/domoticz-yamaha#34]
Any reason why with this running I can't use other software?
With it running, the app on my phone cannot return the data and I can't modify anything like volume, source etc.

**Comment by thomas-villagers** [comment]
Yes: this plugin connects to port 5000, as the iOS/Android App are doing. I fear it is not possible to use both at the same time at the moment (unless we change to the web api...). 

- thomas

**Comment by z1mEk** [comment]
The app on android does not work with the ynca protocol. Uses port 80 and uses XML and both methods work simultaneously. 

I dont know what uses iPhone app.

**Comment by Calzor-Suzay** [comment]
Checked my app and it does look to be using a port just has the IP and YNCA Zone 1.
Shame as it would be handy in Domoticz for reporting/ automated control and in the app for immediate access. I'll watch for any updates :)

**Comment by Calzor-Suzay** [comment]
BTW do you mean this from a quick google?
http://habitech.s3.amazonaws.com/PDFs/YAM/MusicCast/Yamaha%20MusicCast%20HTTP%20simplified%20API%20for%20ControlSystems.pdf

**Comment by z1mEk** [comment]
This is the third control method - REST API;)

**Comment by albebert** [comment]
hum i use the android yamaha app with the domoticz-yamaha plugins without issues.

the only issues i have is #33

**Comment by Calzor-Suzay** [comment]
Lol seems the Amps are pretty much open to any form of connectivity :)
